XCUITests/HomePageSettingsUITest.swift

```diff
@@ -167,4 +168,45 @@ class HomePageSettingsUITests: BaseTestCase {
         // There is one entry
         XCTAssertEqual(app.tables["History List"].cells.count, 3)
     }
+    
+    func testTopSitesCustomNumberOfRows() {
+        var topSitesPerRow:Int
+        let arg = LaunchArguments.LoadDatabasePrefix + prefilledTopSites
```

Comment:
Please do this as part of the set up as in this [example](https://github.com/mozilla-mobile/firefox-ios/blob/master/XCUITests/ActivityStreamTest.swift#L20) otherwise you need to launch the app and terminate it as you are doing but that implies launching the app twice for a single test.